src/main.cpp

```diff
@@ -489,8 +489,9 @@ int main(int argc, char** argv) {
             std::make_unique<RamLoopTransformer>(std::make_unique<RamTransformerSequence>(
                     std::make_unique<HoistAggregateTransformer>(), std::make_unique<TupleIdTransformer>())),
             std::make_unique<ExpandFilterTransformer>(), std::make_unique<HoistConditionsTransformer>(),
-            std::make_unique<CollapseFiltersTransformer>(), std::make_unique<ReorderConditionsTransformer>(),
-            std::make_unique<RamLoopTransformer>(std::make_unique<ReorderFilterBreak>()),
```

Comment:
Why is `ReorderFilterBreak` removed from the pipeline?

Reply:
Indeed - this needs to be fixed.